backend/impl/src/main/java/com/virtuslab/gitmachete/backend/impl/hooks/StatusBranchHookExecutor.java

```diff
@@ -31,7 +31,7 @@
 
 @CustomLog
 public final class StatusBranchHookExecutor extends BaseHookExecutor {
-  private static final int EXECUTION_TIMEOUT_SECONDS = 2;
+  private static final int EXECUTION_TIMEOUT_SECONDS = 5;
```

Comment:
Local tests were still failing when CPU is under load... maybe it should be different for (local) tests and different for production (so that rendering the status doesn't hang too long) 🤔 OTOH, I have no indication that anybody outside of myself actually uses that hooks, in plugin or CLI :trollface:

Reply:
pls share some hooks

Reply:
_share some hooks_? 🤔 as in, provide hook samples in the project directory?